index.bs

```diff
@@ -637,8 +641,49 @@ authorizing an authenticator with which to complete the operation.
         the credential, or that additional information be returned in the attestation statement. Alternatively, the caller may
         specify an additional message that they would like the authenticator to display to the user. Extensions are defined in
         [[#extensions]].
+
+    - The <dfn>attachment</dfn> parameter contains additional constraints on how authenticators may be attached to the
+        client platform.  See [[#attachment]] for a description of the valid attachment values and their meanings.
 </div>
 
+### Credential Attachment enumeration (enum <dfn enum>Attachment</dfn>) ### {#attachment}
+
+<pre class="idl">
+    enum Attachment {
+        "platform",
+        "cross_platform"
+    };
+</pre>
+
+<div dfn-for="Attachment">
+    Clients may communicate with authenticators using a variety of mechanisms.
+    For example, a client may use a platform-specific mechanism to communicate with
+    an authenticator which is physically bound do a platform.  On the other
+    hand, a client may use a variety of transports such as Bluetooth
+    (see [[#transport]] to discover and communicate with authenticators.
+    We define authenticators that are part of the client's platform as having
+    a |platform| attachment, while those that are reachable via standardized
+    cross-platform protocols are defined as having |cross_platform| attachment.
+    This distinction is important because some [RPS] are known to have
+    use-cases where only a |platform| attached authenticator is acceptable (or
+    use-cases where only |cross_platform| attachment may be used).  As a
```

Comment:
wordsmith of L667-L669: 

This distinction is important because there are use-cases where only  |platform| attached authenticators are acceptable to a [RP], and conversely ones where only |cross_platform| attached authenticators are employed.